content/_index.md

```diff
@@ -1,45 +1,55 @@
-    liveText="رویداد شروع شد!"
->}}
+{{< countdown
+title="تا برگزاری رویداد"
+targetDate="2025-10-02T18:00:00+03:30"
+liveText="رویداد شروع شد!"
+
+> }}
 
 {{< info >}}
 
-این یک رویداد فنی نیست! جشن [**«آزادی نرم‌افزار»**](https://digitalfreedoms.org/fa/sfd) فرصتیه که چند ساعتی از مانیتورهاتون فاصله بگیرین و در کنار جمعی از آدم‌های هم‌فازتون هم خوش بگذرونید و هم یاد بگیرید.
+[آزادی نرم‌افزار](https://digitalfreedoms.org/fa/sfd) یک رویداد فنی نیست!
+این جشن فرصتیه که چند ساعتی از مانیتورهاتون فاصله بگیرین و در کنار جمعی از آدم‌های هم‌فازتون
+هم خوش بگذرونید و هم یاد بگیرید.
 
 ما تصمیم داریم امسال هم در لاگ بیرجند دورهمی کوچکی با هدف ترویج و معرفی فلسفه آزادی نرم‌افزار داشته باشیم.
 
 **برای برگزاری بهتر این رویداد به کمک‌های داوطلبانه شما نیاز داریم!**
 
 ### چطور می‌تونم کمک کنم؟
 
-- برای همکاری با تیم اجرایی و انجام هماهنگی‌های لازم از [اینجا](/staff) اعلام حضور کنید.
-- برای حمایت علمی و ارائه درمورد موضوع‌های مرتبط می‌تونید از طریق [فرم ارسال مقاله](/call-for-paper) و مطالعه شرایط، موضوع پیشنهادی‌تون رو ارسال کنید.
-- بخشی از هزینه‌های این رویداد از طریق حمایت‌های مالی داوطلبانه (دونیشن) تامین می‌شه و شما هم می‌تونید برای کمک به تامین این هزینه‌ها [حمایت مالی](/donation) کنید. فهرست حامیان مالی به همراه ریز هزینه‌ها در صفحه [«معرفی حامیان مالی»](/sponsors) درج می‌شه. همین‌طور می‌تونید مشخص کنید که مبلغ واریزی به چه منظور و کجا استفاده بشه.
+- برای همکاری با تیم اجرایی و انجام هماهنگی‌های لازم،
+  از طریق صفحه [ثبت‌نام داوطلبان](/staff) اعلام حضور کنید.
+- اگر مایلید ارائه علمی داشته باشید، می‌تونید از طریق
+  [فرم ارسال مقاله](/call-for-paper) موضوع پیشنهادی‌تون رو بفرستید.
+- برای کمک مالی به برگزاری این رویداد، به صفحه [حمایت مالی](/donation) مراجعه کنید.
+  همچنین فهرست شفاف [حامیان مالی](/sponsors) همراه با ریز هزینه‌ها در دسترسه.
+  شما می‌تونید تعیین کنید که مبلغ واریزی در چه بخشی مصرف بشه.
 
 ### اطلاع‌رسانی کنید!
+
 در هر حالتی می‌تونید از ما حمایت کنید. از معرفی به دوستان‌تون گرفته تا قرار دادن بنرهای زیر در سایت یا وبلاگ‌تون.
+
 - [پوستر در ابعاد کاور شبکه‌های اجتماعی](/image/poster/sfd-social.png)
 - [نسخه قابل چاپ پوستر در ابعاد A5](/image/poster/sfd-a5.png)
-{{< /info >}}
+  {{< /info >}}
 
 {{< card-grid >}}
 {{< card icon="calendar" title="زمان برگزاری" >}}
 پنج‌شنبه، ۱۰ مهر منتظرتون هستیم!
 ساعت برگزاری و زمان‌بندی برنامه‌ها همینجا اعلام می‌شه.
 {{< /card >}}
 {{< card icon="list" title="شرایط حضور" >}}
-در هر سن و سالی که باشید و با هر پیش‌زمینه‌ای می‌تونید به راحتی در این دورهمی شرکت کنید. برای حضور فقط دو نکته رو در نظر داشته باشید:
-لبخند بزنید و لباس گرم رو فراموش نکنید :)
+در هر سنی و با هر پیش‌زمینه‌ای می‌تونید راحت شرکت کنید؛ فقط یادتون باشه لبخند بزنید و لباس گرم بیارید :)
 {{< /card >}}
 {{< card icon="hashtag" title="شبکه‌های اجتماعی" >}}
-مطالب و عکس‌های مربوط به جشن آزادی نرم‌افزار سال ۱۴۰۴ بیرجند رو با هشتگ **#birsfd** در شبکه‌های اجتماعی منتشر کنید. همین‌طور می‌تونید ما رو از شبکه‌های اجتماعی زیر دنبال کنید:
+
+مطالب و عکس‌های جشن آزادی نرم‌افزار ۱۴۰۴ بیرجند رو با هشتگ <b>birsfd</b> به اشتراک بذارید
```

Comment:
better to use markdown here

```diff
- <b>birsfd</b>
+ **birsfd**
```

Reply:
let's explicitly use `#birsfd` here instead of `birsfd` for clarity. what's your opinion?